aries_vcx/src/protocols/proof_presentation/verifier/states/presentation_proposal_received.rs

```diff
@@ -1,14 +1,31 @@
-use messages::protocols::proof_presentation::presentation_proposal::PresentationProposal;
-use messages::protocols::proof_presentation::presentation_request::PresentationRequest;
+use messages::msg_fields::protocols::present_proof::{
+    propose::{PresentationPreview, ProposePresentation, ProposePresentationContent, ProposePresentationDecorators},
+    request::RequestPresentation,
+};
+use uuid::Uuid;
 
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub struct PresentationProposalReceivedState {
-    pub presentation_proposal: PresentationProposal,
-    pub presentation_request: Option<PresentationRequest>,
+    pub presentation_proposal: ProposePresentation,
+    pub presentation_request: Option<RequestPresentation>,
+}
+
+impl Default for PresentationProposalReceivedState {
```

Comment:
It seems we could avoid this Default if on line 31 we set `presentation_request: None` instead of `..Self::default()`

Reply:
Same as the other comment, I'd say it's outside of the PR.